onert-micro/onert-micro/include/pal/mcu/KernelsToBuild.lst

```diff
@@ -58,6 +58,7 @@ REGISTER_KERNEL(MAX_POOL_2D, MaxPool2D)
 REGISTER_KERNEL(MINIMUM, Minimum)
 #/*REGISTER_KERNEL(SHAPE, Shape)*/
 REGISTER_KERNEL(NOT_EQUAL, NotEqual)
+REGISTER_KERNEL(SHAPE, Shape)
```

Comment:
There seems to be the same code (comment out) on line 59.
#/*REGISTER_KERNEL(SHAPE, Shape)*/